okta/users.go

```diff
@@ -566,6 +579,96 @@ func (s *UsersService) Unlock(id string) (*Response, error) {
 	return resp, err
 }
 
+// List User Roles. id must be User.ID
+// will return a struct containing a slice for each role assigned to the user
+// if the user has no roles, return nil
+func (s *UsersService) ListRoles(id string) (*userRoles, *Response, error) {
+
+	u := fmt.Sprintf("users/%v/roles", id)
+	req, err := s.client.NewRequest("GET", u, nil)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	role := make([]userRole, 0)
+	resp, err := s.client.Do(req, &role)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	if len(role) > 0 {
+		myRoles := new(userRoles)
+		for _, v := range role {
+			myRoles.Role = append(myRoles.Role, v)
+		}
+		return myRoles, resp, err
+	}
+
+	return nil, resp, err
+}
+
+// Assign Role to User. id must be User.ID
+func (s *UsersService) AssignRole(id string, role string) (*Response, error) {
+
+	// verify the role is a valid Okta role
+	// https://help.okta.com/en/prod/Content/Topics/Security/Administrators.htm?cshid=Security_Administrators#Security_Administrators
+	allowedRoles := map[string]bool{
```

Comment:
What would be the effect of setting these to `false`? Disable the role at the SDK level?